src/main/java/com/systelab/seed/model/patient/Patient.java

```diff
@@ -1,21 +1,28 @@
 package com.systelab.seed.model.patient;
 
-import com.systelab.seed.model.ModelBase;
-import lombok.AllArgsConstructor;
-import lombok.Data;
-import lombok.EqualsAndHashCode;
-import lombok.NoArgsConstructor;
+import java.io.Serializable;
+import java.util.Date;
 
 import javax.persistence.Embedded;
 import javax.persistence.Entity;
 import javax.persistence.Table;
 import javax.validation.constraints.Size;
-import java.io.Serializable;
-import java.util.Date;
 
-@Entity
+import org.hibernate.envers.Audited;
+
+import com.systelab.seed.model.ModelBase;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.NoArgsConstructor;
+
 @Data
-@EqualsAndHashCode(callSuper = true)
+@Entity
+@Audited
+@Builder 
```

Comment:
Any special reason to use a builder? It's fine if you remove AllArgsConstructor and change the other creation invocation call. If not I prefer to keep it as it was.